### PR TITLE
[master] Fixes up `min` classifier handling

### PR DESCRIPTION
**Pull request by Kortanul**
`min` is a classifier, NOT part of the package name.

this depends on https://github.com/WrenSecurity/wrensec-ui/pull/1 being merged **first**.

### CODE REVIEW

File: forgerock-ui-user/pom.xml [hunk 1]
```diff
@@ -91,13 +97,13 @@
...
+                            <artifactItem>
+                                <groupId>org.forgerock.commons.ui</groupId>
+                                <artifactId>forgerock-ui-commons</artifactId>
+                                <version>${project.version}</version>
+                                <type>zip</type>
+                                <classifier>www</classifier>
+                            </artifactItem>
                         </artifactItems>
                     </configuration>
                     </execution>
```
Comment (siepkes) [inline forgerock-ui-user/pom.xml]: Seems to be a minor indentation issue.

Comment (Kortanul) [reply forgerock-ui-user/pom.xml]: @siepkes take a second look. FR's original indentation was wrong.

Comment (siepkes) [reply forgerock-ui-user/pom.xml]: Ah right, then were good to go!

**Comment by Kortanul** [comment]
@siepkes This got updated. Can you take a second look?

**Pull request merged**